src/audio/copier/copier.c

```diff
@@ -1297,6 +1297,27 @@ static void update_internal_comp(struct comp_dev *parent, struct comp_dev *child
 	child->direction = parent->direction;
 }
 
+static void update_buffer_format(struct comp_buffer __sparse_cache *buf_c,
+				 const struct ipc4_audio_format *in_fmt)
```

Comment:
another nit-pick, can we call this arg just fmt? in one case its input format and in another its output, so better to just keep this simple.

Reply:
sure, thanks!